meinberlin/apps/projectcontainers/templates/meinberlin_projectcontainers/container_list_dashboard.html

```diff
@@ -11,8 +11,10 @@ <h1 class="lr-bar__left">
         {% has_perm 'meinberlin_projectcontainers.add_container' request.user as can_add_project %}
         {% if not can_add_project %}
             <div class="lr-bar__flex--wrapper u-spacer-top">
-                <i class="fas fa-exclamation-circle u-danger u-spacer-right-half u-spacer-top-quarter"></i>
-                <p class="u-danger u-font-italic u-spacer-right">{% trans 'No new project containers can be created. Please use the multimodule feature. Existing project containers can still be edited.' %}</p>
+                <div class="lr-bar__flex--wrapper nowrap">
+                    <i class="fas fa-exclamation-circle u-danger u-icon-spacing"></i>
```

Comment:
The font awesome icons always need something for a11y: either `aria-hidden=true` when they don't have any extra meaning that's not covered by the text. Or an aria-label when you need it, for example for the magnifying glass in a search box.